src/runtime/ExecutionPauser.cpp

```diff
@@ -169,10 +169,29 @@ Value ExecutionPauser::start(ExecutionState& state, ExecutionPauser* self, Objec
             }
             es = new ExecutionState(&state, env, false);
         }
+
+#ifdef ESCARGOT_DEBUGGER
+        Debugger *debugger = state.context()->debugger();
+        bool restoreActiveSavedStackTraceExecutionState = false;
```

Comment:
This bool can be replaced by assigning a special value to activeSavedStackTraceExecutionState, e.g. (void*)0x1. Let me know if you prefer that option.